pkg/dao/repository_configs.go

```diff
@@ -332,6 +361,15 @@ func (r repositoryConfigDaoImpl) Delete(orgID string, uuid string) error {
 
 	if err = r.db.Delete(&repoConfig).Error; err != nil {
 		return err
+	} else {
+		repositoryResponse := api.RepositoryResponse{}
+		ModelToApiFields(repoConfig, &repositoryResponse)
+		//Send delete notification
+		notifications.SendNotification(
+			orgID,
+			notifications.RepositoryDeleted,
+			[]repositories.Repositories{notifications.MapRepositoryResponse(repositoryResponse)},
+		)
 	}
 	return nil
 }
```

Comment:
The previous notification pr sent this in the handler on bulkcreate, i think we should probably send them either in the handler OR in the dao, but not both.  I.e. it should either be at the outer layers or the inner.  Thoughts?

Reply:
Hmm I suppose that makes sense. 👍

Reply:
Thinking on this, it wouldn't make sense to have everything on the handler layer. 
Many dao methods are called in different places. So I think the dao is the logical place to put this, if we want to follow that thought process.